Makefile

```diff
@@ -18,12 +18,21 @@ REGISTRY ?= andyzhangx
 REGISTRY_NAME ?= $(shell echo $(REGISTRY) | sed "s/.azurecr.io//g")
 DRIVER_NAME = disk.csi.azure.com
 IMAGE_NAME ?= azuredisk-csi
+ifndef BUILD_V2
+PLUGIN_NAME = azurediskplugin
 IMAGE_VERSION ?= v1.2.0
+CHART_VERSION ?= latest
+else
+PLUGIN_NAME = azurediskpluginv2
+IMAGE_VERSION ?= v2.0.0-alpha.1
+CHART_VERSION ?= v2.0.0-alpha.1
+GOTAGS += -tags azurediskv2
+endif
 CLOUD ?= AzurePublicCloud
 # Use a custom version for E2E tests if we are testing in CI
 ifdef CI
 ifndef PUBLISH
-override IMAGE_VERSION := e2e-$(GIT_COMMIT)
+override IMAGE_VERSION := $(IMAGE_VERSION)-$(GIT_COMMIT)
```

Comment:
I needed to include the IMAGE_VERSION defined above in the version used by CI in order to get the V2 pipelines to pick up the right build. Otherwise, they were getting the v1 containers.